README.md

```diff
@@ -24,11 +24,11 @@ Icons can be downloaded as SVGs directly from [our website](https://simpleicons.
 Icons can be served from a CDN such as [JSDelivr](https://www.jsdelivr.com/package/npm/simple-icons) or [Unpkg](https://unpkg.com/browse/simple-icons/). Simply use the `simple-icons` npm package and specify a version in the URL like the following:
 
 ```html
-<img height="32" width="32" src="https://cdn.jsdelivr.net/npm/simple-icons@v4/icons/[ICON NAME].svg" />
-<img height="32" width="32" src="https://unpkg.com/simple-icons@v4/icons/[ICON NAME].svg" />
+<img height="32" width="32" src="https://cdn.jsdelivr.net/npm/simple-icons@v4/icons/[ICON SLUG].svg" />
+<img height="32" width="32" src="https://unpkg.com/simple-icons@v4/icons/[ICON SLUG].svg" />
 ```
 
-Where `[ICON NAME]` is replaced by the [icon slug], for example:
+Where `[ICON SLUG]` is replaced by the [icon slug] of the icon you want to use, for example:
```

Comment:
I think, in all cases, "[icon slug]" can now be replaced by simply "[slug]":
```suggestion
Where `[ICON SLUG]` is replaced by the [slug] of the icon you want to use, for example:
```

Reply:
Good one, updated :+1: